en/setup/dev_env_linux.md

```diff
@@ -91,5 +91,10 @@ Note:
 * ROS is installed with Gazebo7 by default (we have chosen to use the default rather than Gazebo8 or Gazebo9 to simplify ROS development).
 * Your catkin (ROS build system) workspace is created at **~/catkin_ws/**.
 
-<!-- import docs for other tools and next steps. -->
-{% include "_addition_dev_tools.txt" %}
+## Additional Tools
+
+After setting up the build/simulation toolchain, see [Additional Tools](../setup/generic_dev_tools.md) for information about other useful tools.
+
+## Next Steps
```

Comment:
Note, pull "Next Steps" out of the additional tools. I think it best to have these clearly labeled and separated.